docs/source/tutorials/open_images_eval/error_analysis/inference/tfhub.py

```diff
@@ -0,0 +1,246 @@
+"""
+Tensorflow Hub Object Detection
+
+Copyright 2017-2020, Voxel51, Inc.
+voxel51.com
+"""
+import csv
+from pathlib import Path
+
+from PIL import Image
+import tensorflow as tf
+import tensorflow_hub as hub
+
+
+class TensorFlowHubDetector:
+    """Object Detector wrapper class that simplifies use of TF Hub models."""
+
+    def __init__(self, model_handle):
+        """Initializes the detector.
+
+        Args:
+            model_handle: url to the model, starting with: https://tfhub.dev/
+        """
+        self.model_handle = model_handle
+        self._detector = hub.load(self.model_handle).signatures["default"]
```

Comment:
After installing multiple tensorflow versions due to various errors, this gave me a permission denied error writing to `/tmp` error from `tensorflow_hub` that finally broke me from trying to generate my own predictions. Not really anything to do here, just venting about tensorflow.

Reply:
"anyone can do data science"